doc/book/helpers/html-tag.md

```diff
@@ -0,0 +1,98 @@
+# HtmlTag
+
+The `HtmlTag` helper is used to **create the root of a HTML document**, the open
```

Comment:
```suggestion
The `HtmlTag` helper is used to **create the root of an HTML document**, the open
```
(Idiomatically, "an" is used before vowel *sounds*, and "a" before consonant sounds. Since we pronounce the letter "H" here, "an" is more appropriate.)